service-config/src/main/java/com/palantir/config/service/ServiceConfiguration.java

```diff
@@ -43,4 +43,23 @@
      * A list of service URIs.
      */
     public abstract List<String> uris();
+
+    // hides implementation details
+    public static Builder builder() {
+        return ImmutableServiceConfiguration.builder();
+    }
+
+    // hides implementation details
+    public interface Builder {
+
+        Builder apiToken(BearerToken apiToken);
+
+        Builder security(SslConfiguration security);
```

Comment:
I am traveling atm, so I don't have the access to link. :(. Want to clarify on the "originalServices" name that the services you set via the builder will not be the same as the services you get from "getServices" because we inject default tokens and security config for service config that don't have them defined. Hence different names. Open to suggestions